### DISCUSSION TITLE
download did not complete at actual file size

### DISCUSSION THREAD

**Discussion by Ahmadrizal894** [skyline-emu/skyline#1642] [General]
![Screenshot_2022-06-30-21-44-50-30_fcf072b9e4efc4ce5ca3e36691a7c3c6.jpg](https://user-images.githubusercontent.com/107229889/176707384-9683faa5-5078-4967-8f53-516a419e1658.jpg)

Fix blur story of seasons

**Comment by PlayGame666** [comment]
What is name game?

**Reply by hope3x** [reply]
It's right there.

**Reply by PlayGame666** [reply]
What is name game

**Comment by Ahmadrizal894** [comment]
not a game, but an emulator file, but now it's safe, looks like a system
problem' github

**Reply by PlayGame666** [reply]
game in the picture

**Reply by Ahmadrizal894** [reply]
> game in the picture

but now it's back to normal...

actually, the game in the picture is the story of season friends of mineral town game... it still looks blurry in the picture, please fix it to provide comfort when playing🙏

**Comment by Ahmadrizal894** [comment]
Sorry, I think I put the wrong image in this message... actually what I
mean is when I download the latest skyline emulator 12mb in size only
finishes at 5mb, it always happens

**Reply by guranu** [reply]
That because the apk is in the downloaded zip file.

**Reply by chostdz** [reply]
It happens to me either but redownload and you should be okay also you might go the site for fresh apk with no zip

**Reply by Dekuzearou** [reply]
It's normal
